packages/apollos-church-api/src/data/campuses/resolver.js

```diff
@@ -3,15 +3,13 @@ import { schemaMerge } from '@apollosproject/server-core';
 import { createAssetUrl } from '../utils';
 
 const resolver = {
+  Query: {
+    campuses: (root, { location }, { dataSources }) =>
+      dataSources.Campus.getPublicByLocation(location),
+  },
   Campus: {
-    latitude: ({ location }) =>
```

Comment:
Why were these properties removed?

Reply:
I don't believe we need them. It just inherits from core. And we don't need all the extra default checking per our conversation from earlier today.